src/test/java/com/cx/restclient/general/AstScaTests.java

```diff
@@ -149,6 +151,14 @@ private void localDirScan(CxScanConfig config) throws MalformedURLException {
 
     private void scanRemoteRepo(String repoUrlProp, boolean useOnPremAuthentication) throws MalformedURLException {
         CxScanConfig config = initScaConfig(repoUrlProp, useOnPremAuthentication);
+
+        RemoteRepositoryInfo repoInfo = new RemoteRepositoryInfo();
```

Comment:
initScaConfig does very similar things. The code here can be made more consistent.